addons/modules/functions/fnc_moduleAmbientFlyby.sqf

```diff
@@ -99,7 +104,7 @@ _waypoint setWaypointCombatMode "BLUE";
 _waypoint setWaypointSpeed _speed;
 
 // Delete aircraft, crew, and group once end waypoint is reached
-_waypoint setWaypointStatements ["true", "private _group = group this; private _aircrafts = []; {if !(vehicle _x in _aircrafts) then {_aircrafts pushback vehicle _x}; deleteVehicle _x} forEach thisList; {deleteVehicle _x;} forEach _aircrafts; deleteGroup _group"];
+_waypoint setWaypointStatements ["true", "private _group = group this; private _aircrafts = []; {_aircrafts pushBackUnique vehicle _x; deleteVehicle _x} forEach thisList; {deleteVehicle _x} forEach _aircrafts; deleteGroup _group"];
```

Comment:
The warning text of deleteVehicleCrew (https://community.bistudio.com/wiki/deleteVehicleCrew) seams interesting for this statement.

Reply:
You can create a new PR if you want to change this, this has already been merged and the branch has been deleted.